v3/_locales/ru/messages.json

```diff
@@ -0,0 +1,299 @@
+  },
+  "options_presets": {
+    "message": "Шаблоны"
+  },
+  "options_custom_jobs": {
+    "message": "Свои задания"
+  },
+  "options_sample": {
+    "message": "Вставить образец"
+  },
+  "options_keywords": {
+    "message": "Поддерживаемые ключевые слова"
+  },
+  "options_events": {
+    "message": "Поддерживаемые события"
+  },
+  "options_desc": {
+    "message": "Переключить описание"
+  },
+  "options_json_desc": {
+    "message": "Вставить список доменов в формате JSON для автоматического обновления после каждого запуска. Если необходимо совпадение URL вместо всего домена, удалите ключ \"hostname\" и добавьте вместо него ключ \"url\". Для получения более подробной информации перейдите на страницу FAQ."
+  },
+  "options_dynamic_json": {
+    "message": "Всегда проверять установку заданий перезагрузки из объекта JSON при загрузке новой страницы."
+  },
+  "options_dynamic_json_desc": {
+    "message": "Если данная настройка включена, расширение устанавливает новое задание перезагрузки для новых страниц, совпадающих с одним из ключей в списке."
+  },
+  "options_policy": {
+    "message": "Политика перезагрузки"
+  },
+  "options_misc": {
+    "message": "Прочее"
+  },
+  "options_reassign": {
+    "message": "Пытаться назначить задание перезагрузки, если на новой вкладке уже выполняется активное."
+  },
+  "options_faqs": {
+    "message": "Открывать страницу FAQs при обновлении"
+  },
+  "options_tools": {
+    "message": "Инструменты"
+  },
+  "options_export_desc": {
+    "message": "Экспорт: Экспортировать все настройки в файл JSON и загрузить его в папку загрузки по умолчанию"
+  },
+  "options_import_desc": {
+    "message": "Импорт: Импортировать настройки из файла JSON (это перезапишет текущие настройки и перезапустит расширение)"
+  },
+  "options_reset_desc": {
+    "message": "Сброс: Сбросить настройки на значения по умолчанию"
+  },
+  "options_reset": {
+    "message": "Сброс значений"
+  },
+  "options_export": {
+    "message": "Экспорт"
+  },
+  "options_import": {
+    "message": "Импорт"
+  },
+  "options_permission": {
+    "message": "Разрешение хоста"
+  },
+  "options_instruction": {
+    "message": "Инструкция по использованию"
+  },
+  "options_ofq": {
+    "message": "Открыть страницу FAQ"
+  },
+  "options_support": {
+    "message": "Поддержать разработку"
+  },
+  "options_save": {
+    "message": "Сохранить настройки"
+  },
+  "options_saved": {
+    "message": "Настройки сохранены"
+  },
+  "options_save_reminder": {
+    "message": "Не забудьте нажать кнопку \"Сохранить настройки\"!"
+  },
+  "bg_reload_tabs": {
+    "message": "Перезагружать вкладки"
+  },
+  "bg_all_tabs": {
+    "message": "Все вкладки"
+  },
+  "bg_reload_all_discarded": {
+    "message": "Все вкладки с отмененной загрузкой"
+  },
+  "bg_reload_window": {
+    "message": "Все вкладки в текущем окне"
+  },
+  "bg_reload_window_discarded": {
+    "message": "Все вкладки с отмененной загрузкой в текущем окне"
```

Comment:
Russian alphabet has 33 letters, including "ё", not 32.

Reply:
Using "ё" is not necessary, and The Ministry of Education of the Russian Federation [confirmed that](https://daily.afisha.ru/news/62111-minprosvescheniya-reshilo-ne-vozvraschat-bukvu-yo-v-pravila-russkoy-orfografii/) recently.

Reply:
> Using "ё" is not necessary, and The Ministry of Education of the Russian Federation [confirmed that](https://daily.afisha.ru/news/62111-minprosvescheniya-reshilo-ne-vozvraschat-bukvu-yo-v-pravila-russkoy-orfografii/) recently.

Use sources of news, not hipsters media, please.

Source of your link: https://www.kommersant.ru/doc/5304275

"Минпросвещения вынесло на общественное обсуждение" means "provided to society discussion" instead of "confirmation", OK?

The source of Kommersant: https://regulation.gov.ru/projects#npa=126402

You can see the current status, it's still in discussion, and I see more dislikes than likes:

![image](https://user-images.githubusercontent.com/6510020/183307184-d09b6e78-46c9-4461-bd3e-290c5df3038e.png)

I don't beleive it'll be accepted. For now.

So, please, according to our current rules, please return the "ё" to your translations. With the all appreciation to your work, I'm just kind of triggered when see "все" vs. "всё", for example, and there are more confusing cases.

Reply:
Unfortunately, likes don't define the likelihood of accepting any draft law in our realities regardless of is media for "hipsters" or not.
Alright, here it is cause I'm not keyboard warrior who wants to waste his own time for fighting for a single letter.

Reply:
> Unfortunately, likes don't define the likelihood of accepting any draft law in our realities regardless of is media for "hipsters" or not.

I agree. But I'd replace "ё" as "йо", for example, because the phonetic and words meaning are different. I'm not against simplification, unless it's confuses and/or not accepted.



> Alright, here it is cause I'm not a keyboard warrior who wants to waste his own time for fighting for a single letter.

Great, I see no offenses with this letter her, thank you very much.